logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -16,6 +16,17 @@
 
 import math
 import json
+import sys
+
+try:
+    import flask
+except ImportError:
+    flask = None
+
```

Comment:
Ah I see I misread it. I would prefer just moving the import into that function, though it's up to maintainers and I forget the repo convention.